docs/articles/documentation/recipes/best-practices/create-faster-tests.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: docs
+title: Create Faster Tests
+permalink: /documentation/recipes/best-practices/create-faster-tests.html
+redirect_from:
+  - /documentation/recipes/create-faster-tests.html
+---
+# Speed Up Your Tests
+
+The speed of TestCafe tests depends on the loading speed of your web application. The best way to speed up your tests may be to optimize the tested application. Make sure that your application doesn't load large payloads of unused CSS or scripts.
```

Comment:
Place ways in such order (from most valuable):

- use the `concurrency` option. Decrease time due to parallelization. It requires more resources because few browser instances will be used.
- use headless browsers. Decrease the test page loading time due to avoiding of the display the rendered page.
- use 'Roles 'feature
- use the fastest speed
- run in local browsers
- mock requests
- optimize your test code (page model, remove unnecessary modules, etc.)

'Optimize your application' is a bad suggestion for the 'Create faster steps' article.